src/framework/auth/hash.clj

```diff
@@ -36,7 +36,7 @@
 (defmethod make :pbkdf2
```

Comment:
we should use buddy.hashers for this :sweat_smile:

Reply:
Is it done? :smile:

Reply:
It will need only to change to a lib and do some refac in future